### DISCUSSION TITLE
Webhooks not triggered after change in external database

### DISCUSSION THREAD

**Discussion by chrisbol** [nocodb/nocodb#4779] [Q&A] [answered]
Just discovering NocoDb, I really like it a lot, great software.
I have connected NocoDb which runs on Cloudron to external Postgres database.
I have just played a bit with the webhook functionality.
This works (f.e. After Insert) like expected if I make changes in the NocoDB UI but when I make a change in the external database the webhook is not triggered.
Is this expected behaviour or is there a way to trigger webhooks after changes in external database?

**Comment by wingkwong** [comment, marked as answer]
External changes won't trigger the webhooks.

**Reply by kairosplan** [reply]
Hello @wingkwong,

Is this the expected behavior? It doesn’t seem to be working for me either.

If this is indeed the expected behavior, perhaps the "Webhooks" tab should be removed when an external database is connected.

Would it be possible to allow webhooks, at least for cases where the external database is modified via the NocoDB API or UI?

Thank you!

**Comment by naimrahidin** [comment]
Dec 2024 and still:

1. The webhooks tab is still visible in the External DB
2. The webhooks are not for External DB
3. There is no update in the doc about this.
